lib/app/utils/terminology.rb

```diff
@@ -145,6 +152,75 @@ def self.load_validators(directory = 'resources/terminology/validators/bloom')
       end
     end
 
+    # Parse the expansions that are in FHIR Models into valueset validators
+    def self.load_fhir_models_expansions
+      Inferno.logger.debug 'Loading FHIR Models Expansions'
+      FHIR::Definitions.expansions.each do |expansion|
+        url = expansion['url']
+        next if @known_valuesets[url]
+        next if SKIP_SYS.include? url
+
+        valueset = Inferno::Terminology::Valueset.new(@db)
+        valueset.valueset_model = FHIR::ValueSet.new(expansion)
+        valueset.vsa = self
+
+        @known_valuesets[valueset.url] = valueset
+
+        bfilter = valueset.generate_bloom
+
+        validate_fn = lambda do |coding|
+          probe = "#{coding['system']}|#{coding['code']}"
+          bfilter.include? probe
+        end
+        # Register the validators with FHIR Models for validation
+        FHIR::DSTU2::StructureDefinition.validates_vs(url, &validate_fn)
+        FHIR::StructureDefinition.validates_vs(url, &validate_fn)
+        @loaded_validators[url] = expansion['expansion']['total']
+      end
+    end
+
+    def self.parse_codesystems_from_valuesets
```

Comment:
This probably won't provide a full expansion of the codesystem, so lets remove this for now.  We'd really have to inspect the compose to make sure the full codesystem is used in order to be sure.

Reply:
Removed